interfaces/oauth_client/v0/README.md

```diff
@@ -0,0 +1,86 @@
+# `oauth_client`
```

Comment:
Why do we need the `_client` suffix? My general standpoint is that interface names should be kept concise and descriptive. Given that the provider of this relation is not even the client, but the oidc provider.